module/applications/item/item-sheet.mjs

```diff
@@ -269,77 +268,6 @@ export default class ItemSheet5e extends ItemSheet {
 
   /* -------------------------------------------- */
 
-  /**
-   * Get the valid item consumption targets which exist on the actor
-   * @returns {Object<string>}   An object of potential consumption targets
-   * @private
-   */
-  _getItemConsumptionTargets() {
```

Comment:
How do we populate consumption targets for uses that are on the Item itself ('base uses') and not on an Activity? Do we call a method on the uses field?

Reply:
Items don't have consumption targets any more. Only activities can consume things now and items just hold uses that can be consumed by activities.

Reply:
Got it, I think that answers some other questions I had too, thanks